.jenkins/cscs-ault/env-hipcc.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+cxx_std="17"
+gcc_version="10.2.0" # Version used for the dependencies
```

Comment:
```suggestion
gcc_version="10.2.0"
```
? It's used for pika as well.